service/src/main/java/org/cbioportal/genome_nexus/service/enricher/HotspotAnnotationEnricher.java

```diff
@@ -57,22 +57,7 @@ public void enrich(VariantAnnotation annotation)
                 hotspotAnnotation.setAnnotation(hotspotsList);
             }
 
-            annotation.setDynamicProp("hotspots", hotspotAnnotation);
-        }
-    }
-
-    private void enrichWithSummary(TranscriptConsequence transcript, List<Hotspot> hotspots)
-    {
-        // add a boolean field to the transcript
-        transcript.setDynamicProp("isHotspot", hotspots.size() > 0);
-    }
-
-    private void enrichWithFullInfo(TranscriptConsequence transcript, List<Hotspot> hotspots)
```

Comment:
Removed, because we no longer add this info directly into a `TranscriptConsequence`